src/scala/com/github/johnynek/bazel_deps/Commands.scala

```diff
@@ -56,6 +56,13 @@ sealed abstract class Command {
 }
 
 object Command {
+  sealed abstract class Gen extends Command {
```

Comment:
`Gen` feels too much like property-based testing's `Gen`. Could we pick some other name like `GenerateLike` please?

Reply:
sounds good.

please take another look.